src/main/java/com/projectkorra/core/game/firebending/blazingarc/BlazingArcAbility.java

```diff
@@ -0,0 +1,68 @@
+package com.projectkorra.core.game.firebending.blazingarc;
+
+import com.google.common.collect.ImmutableSet;
+import com.projectkorra.core.ability.Ability;
+import com.projectkorra.core.ability.AbilityInstance;
+import com.projectkorra.core.ability.AbilityUser;
+import com.projectkorra.core.ability.activation.Activation;
+import com.projectkorra.core.ability.type.Bindable;
+import com.projectkorra.core.skill.Skill;
+import com.projectkorra.core.util.configuration.Configure;
+
+import org.bukkit.event.Event;
+
+public class BlazingArcAbility extends Ability implements Bindable {
```

Comment:
I feel like having both BlazingArcAbility and BlazingArcInstance can get cluttered. Maybe we should just have BlazingArc, a class that extends AbilityInstance, and uses the Ability as a parameter passing it to the AbilityInstance superconstuctor, and generating the BlazingArcAbility from the config or something (like it doesn't need to be its own class that extends Ability; it can be an Ability).